tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-dashboard.html

```diff
@@ -927,21 +991,31 @@ <h2>Continue...</h2>
           // No Session.run() has connected yet. Do not show the spinner(s) or
           // the progress bar(s).
           topRightProgressBar.setAttribute('value', 0);
-          this._setTopRightSpinnerState(false);
+          this._setBusyState(false);
         } else {
           topRightProgressBar.setAttribute('value', progress.value);
-          this._setTopRightSpinnerState(progress.value < 100);
+          this._setBusyState(progress.value < 100);
         }
       },
 
-      _setTopRightSpinnerState(isActive) {
+      _setBusyState(busy) {
```

Comment:
This doesn't quite work because `setTimeout` runs its callback asynchronously, but I like the intention. Lets remove `_setBusyState` and instead have logic directly set `this.set('_busy', [true or false]);`. And then, on the spinner, we set `hidden$=[[!_busy]]`. And `active=[[busy]]`. Apparently, both are necessary. :(

Reply:
Thanks for catching this! This waiting turns out to be unnecessary. I removed this unnecessary calls to setTimeout.

Reply:
Makes sense to me! You could also optionally use an observer to listen for changes to `_busy` and then bind the `active` property to that boolean.